package.json

```diff
@@ -30,17 +30,12 @@
     "headless.js"
   ],
   "scripts": {
-    "start-webpack": "(cd examples/main && yarn && npm run start-local)",
-    "start": "npm run build-watch && budo dist/bundle.js:dist/lumagl.js --live --open --port 3000 --watch-glob '**/*.{html,css,js,glsl}' ",
-    "build-watch": "babel src -d dist --source-maps inline --copy-files --watch & babel examples -d dist --source-maps inline --copy-files --watch &",
+    "start": "cat < docs/get-started/examples.md",
```

Comment:
Probably good to do both 2 & 3 -- PR authors test and reviewers test.
To get back on-track re: this PR tho, agreed (w/ you and @Apercu) that just showing the docs suffices and will save trouble with a complex `npm start` entry point.

However, we now need to make this clear in the root `README` as well. Should probably have been part of this PR in retrospect >.<